skype/skype-ps/skype/Set-CsCceApplianceStatus.md

```diff
@@ -13,9 +13,7 @@ Provide the topic introduction here.
 ## SYNTAX
 
 ```
-Set-CsCceApplianceStatus [[-Identity] <Object>] [-Status <Object>] [-BypassDualWrite <Object>] [-Confirm]
- [-Error <Object>] [-Force] [-Instance <Object>] [-Tenant <Object>] [-WhatIf] [-AsJob] [-Version <Object>]
- [<CommonParameters>]
+Set-CsCceApplianceStatus [[-Identity] <UserIdParameter>] [-Status <Object>] [-Confirm] [-Error <Object>] [-Force] [-Instance <Object>] [-Tenant <Object>] [-WhatIf] [-AsJob] [-Version <Object>] [<CommonParameters>]
```

Comment:
Version and Error are both strings.

In the code, there is also an optional string parameter called ConfigurationSnapshot-- does that not show up when running PlatyPS?

Reply:
Yes It's true, I can see ConfigurationSnapshot in PlatyPS output. I have added it.

This cmdlet is not completed because Ken has decided to continue working on others because build a CCE lab is hard. I think that maybe later we will continue working on it.

Thanks.